### PR TITLE
xfce4-mailwatch-plugin: init at 1.2.0

### PR DESCRIPTION
**Pull request by matthiasbeyer**
###### Motivation for this change

Mailwatch plugin for xfce panel.

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

Pinging xfce ppl here: @vcunat @abbradar @obadz

### CODE REVIEW

File: pkgs/desktops/xfce/panel-plugins/xfce4-mailwatch-plugin.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  name = "${p_name}-${ver_maj}.${ver_min}";
+
+  buildInputs = [ pkgconfig intltool libxfce4util libxfce4ui xfce4panel
+    libxfcegui4 xfconf gtk exo ];
+
+  meta = {
+    homepage = "http://goodies.xfce.org/projects/panel-plugins/${p_name}";
+    description = "Mailwatch plugin for Xfce panel";
+    platforms = platforms.linux;
+    maintainers = [ maintainers.AndersonTorres ];
```
Comment (vcunat) [inline pkgs/desktops/xfce/panel-plugins/xfce4-mailwatch-plugin.nix]: @matthiasbeyer: are you sure about the maintainer?

**Review by vcunat — CHANGES_REQUESTED** [review]

File: pkgs/desktops/xfce/panel-plugins/xfce4-mailwatch-plugin.nix (hunk 1 above)
Comment (matthiasbeyer) [inline pkgs/desktops/xfce/panel-plugins/xfce4-mailwatch-plugin.nix]: Whops, Copy/Paste fail.

**Comment by matthiasbeyer** [comment]
FYI: I'm in the process in building this package into my system, so I can test it.

**Comment by matthiasbeyer** [comment]
Works so far - only tested with local Maildir, because remote IMAP should be encrypted but I cannot toggle this open - will now investigate why.

**Comment by matthiasbeyer** [comment]
Ah, yes. `gnutls` and `libgcrypt` must be in scope.

**Comment by matthiasbeyer** [comment]
BTW, this plugin stores passwords in plaintext in the users home environment. Should we warn users about this? I, personally, have no problem with that because my disks are always encrypted, but others might find that problematic.

**Comment by vcunat** [comment]
IMHO, people that care for this should realize that if an app checks their mail without asking for any kind of password, an adversary can boot the disk and do the same.

**Comment by matthiasbeyer** [comment]
Yes. I have all my machines fully encrypted, so I have no complaint that it stores the password unencrypted... but I'm sure not all users feel this way.

**Comment by vcunat** [comment]
I think Firefox and Thunderbird do the same by default, if you click to save your passwords.  You have to choose to enable "master password" explicitly in the settings.

**Comment by vcunat** [comment]
Thanks for these three plugins.  I pushed them to 17.03 as well (even the same derivation hashes ATM).

**Comment by matthiasbeyer** [comment]
Wow, thank you very much!

**Comment by vcunat** [comment]
:-)

**Pull request merged**